src/plugins/ui/find_and_replace.ts

```diff
@@ -84,13 +85,23 @@ export class FindAndReplacePlugin extends UIPlugin {
       case "ADD_COLUMNS_ROWS":
         this.clearSearch();
         break;
+      case "UPDATE_CELL":
```

Comment:
Unfortunately, this is not sufficient.  the evaluation process can be somehow asynchronous. Some functions can require a remote call to get the data fro the server. In such cases (we do this for odoo functions for instance),  we dispatch the command `EVALUATE_CELLS` that will restart the evaluation, the search should therefore also be updated on `EVALUATE_CELLS`.

this is something worth noting for everyone  @somu-odoo